.gitignore

```diff
@@ -33,7 +33,7 @@ dist
 build-stats.json
 stats.html
 .rpt2_cache
-static-storybook
+storybook-static
```

Comment:
`-static` here but `-preview` elsewhere... is this change related to the other files?

Reply:
It's unrelated, `storybook-static` is the build output which was never added to the gitignore correctly